### ISSUE TITLE
pom.xml has unnecessary <start-class>

### ISSUE THREAD

**Issue by ericdahl** [spring-guides/gs-rest-service#28]
I was confused to see that the ```pom.xml``` includes:

```xml
<properties>
    <start-class>hello.Application</start-class>
</properties>
```

since the [main documentation](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started-first-application-pom) doesn't include references to it. It's unnecessary and adds complexity for a getting started guide.

I was about to submit a pull request to remove it, but then I saw that all of the spring-guides include it. Possibly there's some reason for keeping it?

**Comment by dsyer** [comment]
No reason I know of to have it everywhere. As a learning tool maybe useful to have it once (in the gs-spring-boot guide maybe). I think it's just a remnant.

**Comment by royclarkson** [comment]
It's a remnant.

**Comment by gregturn** [comment]
Resolved via 92403fe70b3a7dd969df51bf2c6c25c08460872a
